pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchSink.java

```diff
@@ -109,6 +109,7 @@ void setElasticsearchClient(ElasticSearchClient elasticsearchClient) {
 
     @Override
     public void write(Record<GenericObject> record) throws Exception {
+        this.elasticsearchClient.incrementCounter(ElasticSearchClient.METRICS_TOTAL_INCOMING, 1);
```

Comment:
This doesn't make much sense - Let's have this metric (actually constant) in `ElasticSearchSink` and just call sinkContext.record from there.